.github/workflows/fuzz.yml

```diff
@@ -0,0 +1,17 @@
+name: CIFuzz
+on: [pull_request]
+jobs:
+  fuzz:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Building and running fuzzers
+      uses: google/oss-fuzz/infra/cifuzz/actions@master
+      with:
+        project-name: 'gonids'
+        fuzz-time: 360
+        dry-run: True
+    - uses: actions/upload-artifact@v1
+      with:
+        name: fuzzer_parserule
```

Comment:
I think these are not fuzz target's name and path, these are for the artifacts created after fuzzing, so would be better to use the values from https://github.com/google/syzkaller/pull/1581/files //cc @leo-neat

Reply:
path: ./out/testcase should fix the failure.
Sorry for the confusion. https://github.com/google/oss-fuzz/pull/3342  will switch the upload artifact from just a single testcase file to as it is above, to a directory containing the testcase files and a file containing the crash stack trace. Is there any other information that you would like to be added to the upload @catenacyber ?

Reply:
>  Is there any other information that you would like to be added to the upload

I do not know what you mean here. I would like to run fuzzing as a regression test against pull requests with the oss-fuzz corpus.